### ISSUE TITLE
Change shared link for resource share

### ISSUE THREAD

**Issue by cjhendrix** [OCHA-DAP/hdx-ckan#4245]
We currently share the link to the resource which will be either:
1. a download from hdx in a goo.gl link
2. a direct link to a remote resource (like a geonode page or dropbox download)

Mixpanel can't count downloads from the 2nd situation.  

Change the link shared for **resource** shares to be the dataset page instead.  

Tagging @alexandru-m-g in case I've misunderstood something.

**Comment by alexandru-m-g** [comment]
What if we share the preview link instead ? I know some resources don't really have a preview but we could imagine a version of the preview page even for those. 
Just seemed a bit weird to share the same link whether you share the whole dataset or just a resource.

**Comment by cjhendrix** [comment]
I agree, it is a bit weird, but our preview page doesn't look very nice. I think it can be @yumiendo or @amcguire62 's call.  Could we perhaps remove the share links from resources?

**Comment by yumiendo** [comment]
I find resource level 'share' lacking bit of information/credentials (user receives a direct link to download and its missing description, meta data, source, etc)

my preference would be to remove the share option from resources and we let users 'share' at dataset level only... unless @amcguire62 thinks this is a deal breaker :)

In addition - shall we add a bit of details in the 'share' email:

Title of dataset (via HDX) <- in the email title
- include the first 25 words(or so) of description
- a goo.gl link

example:
Lake Chad Basin FTS Appeal Data (via HDX) 
Contains data from OCHA's Financial Tracking Service on the financial requirements and current funding levels for appeals in the Lake Chad Basin crisis countries.
http://goo.gl/5o5LCb

**Comment by cjhendrix** [comment]
Sounds good to me!

**Comment by amcguire62** [comment]
+1 good for me also

**Comment by alexandru-m-g** [comment]
I have pushed a change for removing the share link for resources.

@yumiendo @danmihaila 
About the second change related to the email body, without changing too much the mechanism for how that works we could do something like the following:
-  here is the test dataset with a long formatted description 
  ![description](https://cloud.githubusercontent.com/assets/6586348/16357248/8e4e15d8-3af9-11e6-8741-417bb209bb64.png)
-  and here is how the email would look like
  ![description2](https://cloud.githubusercontent.com/assets/6586348/16357257/e80f3e6c-3af9-11e6-86e8-4eb3f48c4fcf.png)

Notice that the formatting is gone. To keep it we should probably change the the way this works. Right now the email is actually sent by the user from his email client that automatically opens. To have a nice formatting we would need to send the email from our servers and provide a nice web form for that.
